DigitalEcosystem/exploratory/initial_work/initial_bg_work_commentated/bg_extra_descriptors.py

```diff
@@ -1,18 +1,35 @@
 #!/usr/bin/env python
```

Comment:
We decided to instead keep a different file for the bandgap initial work. This file is commented, but it was more appropriate to move this work into this branch, since it's really just another file we're moving.

Reply:
I don't understand what you said above. Do you mean we decided to focus on a different "refined" notebook instead of this one, although this one is commented?

Reply:
I provided commentary for this notebook before we restructure which notebooks would wind up in "refined." Since it's just another file we moved, it's in this PR.